### ISSUE TITLE
`no-invalid-token-paths` reports false positive with color opacity modifier

### ISSUE THREAD

**Issue by oneHoopoe** [chakra-ui/eslint-plugin-panda#253]
Given this code:

```tsx
import { styled } from '../styled-system/jsx'

function App() {
  return (
    <styled.p css={{boxShadow: "0px 5px 5px {black/25}"}}>Reproduction</styled.p>
  )
}

export default App
```

ESlint reports a following error:

```
5:32  error  `black/25` is an invalid token path  @pandacss/no-invalid-token-paths
```

I have created a [reproduction repo](https://github.com/oneHoopoe/pandacss-eslint-repro) as a minimum reproducible example, if it helps. The example CSS rule is in [App.tsx](https://github.com/oneHoopoe/pandacss-eslint-repro/blob/master/src/App.tsx).

**Comment by anubra266** [comment]
@oneHoopoe this is because black is not a token path, so it gets interpreted as a raw color. if you're using the token syntax, it should be `colors.black` not `black`

Note the difference in the outputs:

![Image](https://github.com/user-attachments/assets/7c8bd1a4-163d-4aca-bbbb-f2e9c96844d5)
